src/main/java/htsjdk/samtools/cram/compression/rans/Utils.java

```diff
@@ -34,19 +34,110 @@ static void reverse(final ByteBuffer byteBuffer) {
     }
 
     // Returns the current cumulative frequency (map it to a symbol yourself!)
-    static int RANSDecodeGet(final int r, final int scaleBits) {
+    public static int RANSDecodeGet(final int r, final int scaleBits) {
```

Comment:
If this returns the cumulative frequency, let's rename it to `getCumulativeFrequency`. (Likewise `r` could use a better name - its the byte for the nth interleave window right ? Maybe `ransByte`.)